DEVELOPMENT.md

```diff
@@ -24,7 +24,15 @@ eas "$@" # or $HOME/expo/eas-cli/bin/run "$@"
 
 #### Introducing breaking changes for job API (@expo/eas-build-job package)
 
-If you want to intruduce breaking changes to the `@expo/eas-build-job` package contact one of the [CODEOWNERS](/CODEOWNERS), so we can make sure that legacy cases are still suported on EAS build servers and in GraphQL API. Please describe what changes you want to make and why and we will implement api changes that support both cases. After everything is deployed to production you can introduce PR that relies on the new implementation.
+If you want to introduce breaking changes to the `@expo/eas-build-job` package contact one of the [CODEOWNERS](/CODEOWNERS), so we can make sure that legacy cases are still supported on EAS build servers and in GraphQL API. Please describe what changes you want to make and why and we will implement API changes that support both cases. After everything is deployed to production you can introduce PR that relies on the new implementation.
 
 
 Change like that could be e.g. introducing a new required field in the job object. In that case, we would make changes to allow this field and provide default values for legacy requests, but it would be ignored by the build process. After everything is deployed/published, your follow-up PR would introduce logic that is using this new field.
```

Comment:
```suggestion
A breaking change like that could be introducing a new required field on the job object. In that case, we would make changes to allow this field and provide default values for legacy requests, but it would be ignored by the build process. After everything is deployed/published, your follow-up PR would introduce logic that is using this new field.
```